elf.h

```diff
@@ -47,7 +47,7 @@ bool elf_open(elf_t *e, const char *path);
 /* Find a symbol entry */
 const struct Elf32_Sym *elf_get_symbol(elf_t *e, const char *name);
 
-/* Find symbole from a specified ELF file */
+/* Find symbol from a specified ELF file */
```

Comment:
Don't change `elf.h` within this pull request. Instead, change in another pull request.